swagger.yaml

```diff
@@ -721,6 +721,30 @@ paths:
                 items:
                   $ref: "#/components/schemas/Error"
 
+  "/severa/flextime/{severaGuid}": 
+    get:
+      tags:
+        - UsersRead
+      summary: 
+        Get the flextime balance for a user. Total balance
+        is returned for the current date. Month balance is the balance for the
+        month of the current date.
+      operationId: getFlextimeByUser
+      parameters:
+        - name: severaGuid
+          in: path
+          required: true
+          description: severaGuid of the user.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Flextime entry retrieved successfully
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Flextime"
```

Comment:
There should also be a default error response as was done for questionnaries:
 "default":
            description: Invalid request was sent to the server.
            content:
              application/json:
                schema:
                  type: array
                  items:
                    $ref: "#/components/schemas/Error"